lib/worker/subprocess.js

```diff
@@ -137,6 +137,11 @@ ipc.options.then(async options => {
 	const load = async ref => {
 		for (const extension of extensionsToLoadAsModules) {
 			if (ref.endsWith(`.${extension}`)) {
+				try {
+					return await import(ref); // eslint-disable-line no-await-in-loop
+				} catch {
```

Comment:
This swallows any errors thrown in the `ref` module itself.

Instead you should detect whether `import()` works and check that before loading `ref`. Then you can send the appropriate "ESM is not available in this Node.js version" kinda error, or have AVA report the error from the module.

Reply:
I am having difficulties to understand the windows problem.
I would like to see the error coming out of the CI build.